### PR TITLE
Added a greeting for Biruk.

### PR DESCRIPTION
**Pull request by biruk741**
This one makes it print my name with a greeeting.

### CODE REVIEW

File: src/main/java/hellos/Hellos.java [hunk 1]
```diff
@@ -19,12 +19,17 @@ public void run() {
...
 
+    builder.append(birukSaysHello());
     builder.append(kkSaysHello());
     builder.append(nicSaysHello());
 
     return builder.toString();
   }
 
+  private String birukSaysHello() {
+    return "Biruk says 'Yo!'\n";
```
Comment (NicMcPhee) [inline src/main/java/hellos/Hellos.java:30]: Dude, "Yo!" isn't a very professional greeting.

**Review by NicMcPhee — DISMISSED** [review]
This looks awesome – thanks for the work!

File: src/main/java/hellos/Hellos.java (hunk 1 above)
Comment (lam00071) [inline src/main/java/hellos/Hellos.java:61]: I mean, yo hahahahahahahagah

**Review by lam00071 — DISMISSED** [review]
YooOoOooooO HAGAGAGAGA. Great code ! :D

**Review by LloydHilsgen001 — DISMISSED** [review]
Looks good!

**Comment by NicMcPhee** [comment]
Not that your name for this pull request ("Added a greeting for myself") isn't super helpful when looking at a long list of PR titles since it's not clear who "myself" refers to. I would change that to be a specific reference to your name.

File: src/main/java/hellos/Hellos.java [hunk 2]
```diff
@@ -19,6 +19,8 @@ public void run() {
   public String generateOutput() {
     StringBuilder builder = new StringBuilder();
 
+
```
Comment (NicMcPhee) [inline src/main/java/hellos/Hellos.java]: I probably wouldn't have two blank lines here.

**Review by NicMcPhee — CHANGES_REQUESTED** [review]
Note that your name for this pull request ("Added a greeting for myself") isn't super helpful when looking at a long list of PR titles since it's not clear who "myself" refers to. I would change that to be a specific reference to your name.

**Review by NicMcPhee — DISMISSED** [review]

**Review by IsabelleHjelden — DISMISSED** [review]
Looks good!

**Review by JacobJenness — APPROVED** [review]

**Pull request merged**